test/functional/update/update-to-same-version.bats

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bats
+
+load "../testlib"
+
```

Comment:
please consider adding your contact info at the top of the test for consistency with the rest of the other tests:

```
# Author: Otavio Pontes
# Email: otavio.pontes@intel.com
```
We started doing this as a good practice in tests so it is easier to find the author and ask questions if needed.

Reply:
Fixed that, but do we really need this? Git keeps the history of that file and git blame is better than this info to find who to talk to.

Reply:
well, consider this example; when I migrated all the tests from the old framework to the new, all old tests were deleted so if you look at all new tests they show as if I'm the author (which in a way I guess I am) but all those tests were actually designed by someone else, I just ported them to the new framework. So in that scenario we lost traceability of the author of all those tests. When I was porting the tests I had a lot of questions about them and when I looked at GitHub to see who the author was I ran into the problem that sometimes the person who had submitted the test was not the author or things like that so it was very difficult to find who to ask for each test.
So I guess is just extra info that could be useful, and it doesn't harm if it is there.

(If you use the generate_test function from the testilb.bash to automatically create a new blank test it will already add your contact info to the test)